### PR TITLE
fix wrong recovery percentage

### PR DESCRIPTION
**Pull request by hassnaaHamdi**
count_d is used instead of count_r during calculating recovery percentage for women and men.

### CODE REVIEW

File: data.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def percentage(part, whole):
     if(i == 'Died'):
         count_d = count_d + 1
     elif(i == 'Recovered'):
-        count_r = count_d + 1
+        count_r = count_r + 1
```
Comment (Asher8118) [inline data.py:101]: Leaving a review comment

**Review by Asher8118 — CHANGES_REQUESTED** [review]
Can you address these comments?

File: data.py (hunk 1 above)
Comment (hassnaaHamdi) [reply data.py:101]: Resolved.

**Review by Asher8118 — APPROVED** [review]
LGTM!

**Pull request merged**